src/ci/proxy.sh

```diff
@@ -12,9 +12,11 @@ mkdir -p artifacts/proxy
 cd src/proxy-manager
 cargo fmt -- --check
 cargo clippy --release -- -D warnings
-# RUSTSEC-2020-0016: a dependency net2 (pulled in from tokio) is deprecated
-# RUSTSEC-2021-0065: a dependency anymap is no longer supported
-cargo audit --deny warnings --deny unmaintained --deny unsound --deny yanked --ignore RUSTSEC-2020-0016 --ignore RUSTSEC-2021-0065
+# RUSTSEC-2020-0016: a dependency `net2` (pulled in from `tokio`) is deprecated
```

Comment:
I suggest you update the target PR (feature branch) with these unrelated changes before merging, to keep the feature branch's history easy to understand.